content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -57,58 +60,65 @@ For example:
 
 1. List all of the pods in a given namespace.
 
-        GET /api/v1/namespaces/test/pods
-        ---
-        200 OK
-        Content-Type: application/json
-        {
-          "kind": "PodList",
-          "apiVersion": "v1",
-          "metadata": {"resourceVersion":"10245"},
-          "items": [...]
-        }
+   ```console
+   GET /api/v1/namespaces/test/pods
+   ---
+   200 OK
+   Content-Type: application/json
+   {
+     "kind": "PodList",
+     "apiVersion": "v1",
+     "metadata": {"resourceVersion":"10245"},
+     "items": [...]
+   }
+   ```
 
 2. Starting from resource version 10245, receive notifications of any creates, deletes, or updates as individual JSON objects.
 
-        GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245
-        ---
-        200 OK
-        Transfer-Encoding: chunked
-        Content-Type: application/json
-        {
-          "type": "ADDED",
-          "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "10596", ...}, ...}
-        }
-        {
-          "type": "MODIFIED",
-          "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "11020", ...}, ...}
-        }
-        ...
+   ```
+   GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245
+   ---
+   200 OK
+   Transfer-Encoding: chunked
+   Content-Type: application/json
+   {
```

Comment:
mini nit: HTTP/1.1 usually has a blank line between headers and body; I'd include that here (it looks odd to me without it)